### PR TITLE
added retrieving vetext api token from param store

### PR DESCRIPTION
**Pull request by jakehova**

### CODE REVIEW

File: lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def vetext_incoming_forwarder_lambda_handler(event: any, context: any):
 def get_body_from_sqs_invocation(event):
     event_bodies = []
 
-    try: 
+    try:
```
Comment (kalbfled) [inline lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py:83]: I assume this is accidental white space.

**Review by jessecanderson — APPROVED** [review]

File: lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py [hunk 2]
```diff
@@ -113,14 +113,30 @@ def get_body_from_alb_invocation(event):
...
+    boto_client = boto3.client('ssm')
+    
+    resp = boto_client.get_parameter(
+        Name=f"{key}",
+        WithDecryption=True
+    )
+
+    logging.info(resp)
+
+    return resp["Parameter"]["Value"]
```
Comment (kalbfled) [inline lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py:126]: You could make this more error resistant by catching KeyError is doing something like this:
```
return response.get("Parameter", {}).get("value", '')
```

File: lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py [hunk 3]
```diff
@@ -113,14 +113,30 @@ def get_body_from_alb_invocation(event):
...
+
 def make_vetext_request(request_body):
     try:
         connection = http.client.HTTPSConnection(os.environ.get('vetext_api_endpoint_domain'),  context = ssl._create_unverified_context())
 
         # Authorization is basic token authentication that is stored in environment.
+        authToken = read_from_ssm(os.environ.get('vetext_api_auth_ssm_path'))
```
Comment (kalbfled) [inline lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py:133]: You could catch KeyError here or maybe use a sensible default.

**Review by kalbfled — CHANGES_REQUESTED** [review]
Please remove the no-op try-except blocks.  I commented on one, but there are others in the same file.

**Review by kalbfled — APPROVED** [review]

**Pull request merged**